### DISCUSSION TITLE
CryptoMining also?

### DISCUSSION THREAD

**Discussion by Damienb2022** [martinet101/ElevenClock#845] [Q&A]
Hey random question. I had an error appear with a python script that pointed me to %appdata%/local/Temp/_MEI200842.

After some digging, in this folder are a load of python extensions and dlls. Resources with the icons for ElevenClock. What i find interesting is the particular dll: libcrypto and a python referencing Shiboken. If i quit ElevenClock the entire folder disappears

Does ElevenClock do crypto mining in the background?

![pythonerror](https://user-images.githubusercontent.com/99514170/180965812-b60d7aa3-147e-40f5-b605-8c937388790a.png)
.

**Comment by marticliment** [comment]
> Does ElevenClock do crypto mining in the background?

No, it does not!



> libcrypto

This dll is used by python to handle certificates. Since ElevenClock connects to the internet to checkj for updates/load announcements over https, this is needed

**Reply by marticliment** [reply]
Shiboken is a Qt module that I don't really know what it does, but it is related to binding ftom Python to c++

**Comment by marticliment** [comment]
If this error still appears, please change this in the settings:
![image](https://user-images.githubusercontent.com/53119851/182141725-4b49b8c1-3847-4495-b6ec-588fb77d61b8.png)
